pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -41,7 +42,7 @@
           = view :display_video_thumbnail, thumbnail_only: true, id: "mainvideo", video_code: entry[:youtube_id], image_url: entry[:image_url], play_button: 'center', letterbox: "false"
       - elsif entry[:type] == "text"
         .text{dir: "auto"}
-          = I18n.t(entry[:text])
+          != I18n.t(entry[:text], markdown: :inline)
```

Comment:
I think this should either be

        .text{dir: "auto"}
          != I18n.t(entry[:text], markdown: true)

or

        %p.text{dir: "auto"}
          != I18n.t(entry[:text], markdown: :inline)

Reply:
Oh interesting.  We tried the former and saw that it generated wrapper `<p>` tags but they weren't needed for our layout.  Are they important from a security point of view?

Reply:
from an HTML validity point; `div`s should contain only flow content like paragraphs, not phrasing content like text

Reply:
Ah fair enough.  I'll go back to the former then!

Reply:
oh, also: using the `:inline` option will only render the subset of markdown syntax that can validly be rendered inline, so no lists, headers, etc. Presumably not a problem for this content, but still a potential source of confusion